### ISSUE TITLE
Uncaught Invariant Violation: Minified React error #149; visit http://facebook.github.io/react/do...

### ISSUE THREAD

**Issue by nilanjansiromani** [facebook-atom/atom-ide-ui#123]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.22.1 x64
**Electron**: 1.6.15
**OS**: Mac OS X 10.13
**Thrown From**: [atom-ide-ui](https://github.com/facebook-atom/atom-ide-ui) package 0.6.0

### Stack Trace

Uncaught Invariant Violation: Minified React error #149; visit http://facebook.github.io/react/docs/error-decoder.html?invariant=149

**Comment by hansonw** [comment]
Hey - could you provide the full stack trace or a screenshot? thanks!

**Comment by moranje** [comment]
I don't know about his but mine is:

```
Invariant Violation: Minified React error #149; visit http://facebook.github.io/react/docs/error-decoder.html?invariant=149&args[]=input for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at w (/Users/martien/Dropbox/dotfiles/apps/.atom/packages/atom-ide-ui/node_modules/react-dom/cjs/react-dom.production.min.js:12:224)
    at Me (/Users/martien/Dropbox/dotfiles/apps/.atom/packages/atom-ide-ui/node_modules/react-dom/cjs/react-dom.production.min.js:124:445)
    at B (/Users/martien/Dropbox/dotfiles/apps/.atom/packages/atom-ide-ui/node_modules/react-dom/cjs/react-dom.production.min.js:129:479)
    at Ca (/Users/martien/Dropbox/dotfiles/apps/.atom/packages/atom-ide-ui/node_modules/react-dom/cjs/react-dom.production.min.js:132:473)
    at /Users/martien/Dropbox/dotfiles/apps/.atom/packages/atom-ide-ui/node_modules/react-dom/cjs/react-dom.production.min.js:137:206
    at g (/Users/martien/Dropbox/dotfiles/apps/.atom/packages/atom-ide-ui/node_modules/react-dom/cjs/react-dom.production.min.js:147:110)
    at f (/Users/martien/Dropbox/dotfiles/apps/.atom/packages/atom-ide-ui/node_modules/react-dom/cjs/react-dom.production.min.js:147:42)
    at beginWork (/Users/martien/Dropbox/dotfiles/apps/.atom/packages/atom-ide-ui/node_modules/react-dom/cjs/react-dom.production.min.js:151:403)
    at e (/Users/martien/Dropbox/dotfiles/apps/.atom/packages/atom-ide-ui/node_modules/react-dom/cjs/react-dom.production.min.js:182:356)
    at k (/Users/martien/Dropbox/dotfiles/apps/.atom/packages/atom-ide-ui/node_modules/react-dom/cjs/react-dom.production.min.js:183:362)
    at p (/Users/martien/Dropbox/dotfiles/apps/.atom/packages/atom-ide-ui/node_modules/react-dom/cjs/react-dom.production.min.js:184:367)
    at y (/Users/martien/Dropbox/dotfiles/apps/.atom/packages/atom-ide-ui/node_modules/react-dom/cjs/react-dom.production.min.js:188:412)
    at D (/Users/martien/Dropbox/dotfiles/apps/.atom/packages/atom-ide-ui/node_modules/react-dom/cjs/react-dom.production.min.js:187:423)
    at Object.enqueueSetState (/Users/martien/Dropbox/dotfiles/apps/.atom/packages/atom-ide-ui/node_modules/react-dom/cjs/react-dom.production.min.js:140:182)
    at VirtualizedFileTree.v.setState (/Users/martien/Dropbox/dotfiles/apps/.atom/packages/atom-ide-ui/node_modules/react/cjs/react.production.min.js:12:357)
    at VirtualizedFileTree._processStoreUpdate (/Users/martien/Dropbox/dotfiles/apps/.atom/packages/nuclide/pkg/nuclide-file-tree/components/VirtualizedFileTree.js:283:10)
    at _disposables.add._store.subscribe (/Users/martien/Dropbox/dotfiles/apps/.atom/packages/nuclide/pkg/nuclide-file-tree/components/VirtualizedFileTree.js:192:60)
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at a._animationFrameRequestSubscription.nextAnimationFrame.subscribe [as _next] (/Users/martien/Dropbox/dotfiles/apps/.atom/packages/nuclide/pkg/nuclide-file-tree/lib/FileTreeStore.js:605:21)
    at a.__tryOrUnsub (/Users/martien/Dropbox/dotfiles/apps/.atom/packages/atom-ide-ui/node_modules/rxjs/bundles/Rx.min.js:43:337)
    at a.next (/Users/martien/Dropbox/dotfiles/apps/.atom/packages/atom-ide-ui/node_modules/rxjs/bundles/Rx.min.js:42:227)
    at a._next (/Users/martien/Dropbox/dotfiles/apps/.atom/packages/atom-ide-ui/node_modules/rxjs/bundles/Rx.min.js:40:393)
    at a.next (/Users/martien/Dropbox/dotfiles/apps/.atom/packages/atom-ide-ui/node_modules/rxjs/bundles/Rx.min.js:40:64)
    at requestAnimationFrame (/Users/martien/Dropbox/dotfiles/apps/.atom/packages/atom-ide-ui/modules/nuclide-commons/observable.js:372:14)
```

**Comment by semmypurewal** [comment]
@hansonw Looks like there's more info here:

https://github.com/facebook/nuclide/issues/1386

Seems related to file tree.

**Comment by hansonw** [comment]
@moranje Do you have both atom-ide-ui and Nuclide activated? I wonder if this goes away after you deactivate one of them. On a side note Atom does load a different React version as part of the GitHub package so maybe we need to rectify that somehow.

**Comment by hansonw** [comment]
Should be fixed in 33444edc4b56b925ec7a88e2e61e8f1f6cbdaada (releasing soon).
